conda/recipes/libkvikio/recipe.yaml

```diff
@@ -99,6 +101,7 @@ outputs:
                 - libcufile ${{ cuda11_libcufile_run_version }}
                 - libcufile-dev ${{ cuda11_libcufile_run_version }}
           else:
+            - cuda-nvml-dev
```

Comment:
So this NVML package only has headers and a stub library

The driver has a library, but it is not in this package

If we are building a KvikIO library, there should be no need for this dependency at runtime